articles/openshift/oshift-trials-gs.md

```diff
@@ -0,0 +1,67 @@
+
+## Introduction to UKCloud for OpenShift trials
+
+OpenShift is a platform as a service, this means that you no longer need to worry about the underlying hardware or operating system, instead you can focus on developing, deploying, scaling and monitoring your applications in OpenShift which maximises your returns on investment both monetary and time.
+
+With UKCloud OpenShift you have admin access to OpenShift giving you the control you need to create, deploy and administer your cluster as you see fit.
+
+All OpenShift deployments get a starter pack which includes the management and orchestration planes of OpenShift. These are essentially 3 Master nodes, 2 Infrastructure nodes and load balancers for the control and data planes.
+
+During a trial each cluster is also provisioned with 2 small run time packs. These are essentially 2 OpenStack VM's running RedHat atomic OS with 16GB RAM, 2vCPU's and 100GB disks each. This is also the minimum number of Runtime Packs UKCloud recommends in each OpenShift deployment in order to provide minimal resilience and enable interruption free patching. After the trial phase it is possible to deploy larger and more nodes for your cluster.
+
+**OpenShift documentation:** [*Architecture*](https://docs.openshift.com/container-platform/3.11/architecture/index.html)
+
+## GUI and CLI Access
+
+In order to access the OpenShift cluster GUI customers require a internet-enabled device running a HTML5 capable browser and need to be set up for access in the UKCloud Portal. Customers will also be set up with the relevant access in their OpenShift deployment.
+
+**OpenShift documentation:** [*Web Console Walkthrough*](https://docs.openshift.com/container-platform/3.11/getting_started/developers_console.html)
+
+In addition to GUI access customers can also access OpenShift through the CLI, in order to do this customers need to have installed the OC client on their machines. 
+
+**OpenShift documentation:** [*Installing the OC CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#installing-the-cli)
+
+## Storage
+
+Each trial deployment has a block storage quota of 2TB across either tier 1 or tier 2 storage. This is enforced at the OpenStack project layer. The storage is available to the projects customers create within OpenShift, customers can consume that storage by creating Persistent Volume Claims (PVCs) for their applications. It is good practice to use PVCs for any persistent storage requirements as any data stored within local volumes will be lost when pods are moved from one node to another; for example as a result of node evacuations during patching. Storage quotas can be increased by creating a service request in the UKCloud portal.
```

Comment:
You can increase your storage quotas by creating a service request in the UKCloud Portal.